test/gurka/test_use.cc

```diff
@@ -125,10 +126,10 @@ TEST_F(Use, test_entering_rest_area) {
 
   // Verify the second maneuver instructions
   gurka::assert::raw::expect_instructions_at_maneuver_index(
-      result, step_idx, "Turn left toward Bear Peak.",
-      "Turn left toward Bear Peak. Then You will arrive at your destination.",
-      "Turn left toward Bear Peak.",
-      "Turn left toward Bear Peak. Then You will arrive at your destination.",
+      result, step_idx, "Turn left toward Bear Peak Rest Area.",
```

Comment:
these expected results seem odd - I would try to mark all the edges with oneway - to be closer to real world. I know we still have more work to for Rest/Service Areas Use to make them work as exits instead of forks

Reply:
Hmm.  I trust what you say is true.  However, I didn't change this behavior.  I'm just giving more real world names to the rest-area and service-area in this test.

Reply:
@ktatterso we will fix later - thanks